.storybook/config.js

```diff
@@ -1,8 +1,9 @@
 import { configure } from '@storybook/react';
 import { addDecorator } from '@storybook/react';
 import StoryRouter from 'storybook-react-router';
+import requireContext from 'require-context.macro';
```

Comment:
Interesting, I haven't used this before. What does this provide over the existing?

Reply:
https://github.com/storybookjs/storybook/tree/next/addons/storyshots/storyshots-core#configure-jest-to-work-with-webpacks-requirecontext. 

> babel-plugin-macros defines a standard interface for libraries that want to use compile-time code transformation without requiring the user to add a babel plugin to their build system (other than babel-plugin-macros, which is ideally already in place).

Macros are an alternative to babel plugins, and in our case allow us to avoid ejecting create-react-app. You can read more here as well: https://github.com/kentcdodds/babel-plugin-macros#this-solution